src/main/java/org/opentripplanner/common/geometry/GeometryUtils.java

```diff
@@ -153,6 +156,23 @@ else if (r > 1.0)
         return r;
     }
 
+
+    /**
+    */
+    public static double calculateDistance(Coordinate from, Coordinate to) {
+        try {
+            // the following code is based on JTS.orthodromicDistance( start, end, crs )
+            GeodeticCalculator gc = new GeodeticCalculator(WGS84_XY);
+            gc.setStartingPosition(JTS.toDirectPosition(from, WGS84_XY));
+            gc.setDestinationPosition(JTS.toDirectPosition(to, WGS84_XY));
+
+            return gc.getOrthodromicDistance();
+        }
+        catch (TransformException e) {
+            throw new IllegalStateException(e.getMessage(), e);
+        }
+    }
+
```

Comment:
Doc:
https://upload.wikimedia.org/wikipedia/commons/3/3e/WGS84_mean_Earth_radius.svg
Hannes Junnila12:07
https://www.cambridge.org/core/journals/journal-of-navigation/article/sphere-to-spheroid-comparisons/E757B940C8C622A064276AC33CDC15C0
https://www.cambridge.org/core/journals/journal-of-navigation/article/sphere-to-spheroid-comparisons/E757B940C8C622A064276AC33CDC15C0/core-reader